locales/en/filters.json

```diff
@@ -150,5 +150,9 @@
 	{
 		"hostlist.39.name": "The Blocklist Project - Ads List",
 		"hostlist.39.description": "The collection of web addresses that have been identified as hosting or serving unwanted advertisements."
+	},
+	{
+		"hostlist.40.name": "TUR: Turkish Ad Hosts",
+		"hostlist.40.description": "Necessary hosts file to block tracking (and possibly advertising) networks in commonly used mobile applications (and possibly WEB pages) in Turkey."
```

Comment:
do not forget to upload the base strings to crowdin

Reply:
@Sergey-Lyapin so what about this?